bgpd/bgp_evpn.c

```diff
@@ -5598,6 +5649,22 @@ int bgp_evpn_local_es_add(struct bgp *bgp,
 	return 0;
 }
 
+/*
+ * Handle change in setting for BUM handling. The supported values
+ * are head-end replication and dropping all BUM packets. Any change
+ * should be registered with zebra. Also, if doing head-end replication,
+ * need to advertise local VNIs as EVPN RT-3 wheras, if BUM packets are
+ * to be dropped, the RT-3s must be withdrawn.
+ */
+void bgp_evpn_flood_control_change(struct bgp *bgp)
+{
```

Comment:
Suggestion: A debug message here indicating enable/disable action would help in debugging. Atleast to know what time it is disabled/enabled etc.

Reply:
added a zlog_info since this doesn't happen a bunch.